src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -76,54 +96,113 @@ export const MyFieldAndBackground = () => {
-                    </>
-                  )}
-                </Field>
-              </Box>
-              <Box sx={{ display: 'flex', justifyContent: 'end', mt: '1rem' }}>
-                <LoadingButton loading={isSubmitting} disabled={!dirty} variant="contained" type="submit">
-                  {t('common.save')}
-                </LoadingButton>
-              </Box>
-            </Form>
+                    )}
+                  </Field>
+                  <Typography variant="body1" fontStyle={'italic'} sx={{ mb: '2rem' }}>
+                    {t('my_page.my_profile.field_and_background.keywords_search_text')}
+                  </Typography>
+                </Box>
+
+                <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', mt: '4rem' }}>
+                  <Typography variant="h2" sx={{ mb: '1rem' }}>
+                    {t('my_page.my_profile.background')}
+                  </Typography>
+                  <Typography variant="h3">{t('my_page.my_profile.field_and_background.norwegian')}</Typography>
+                  <Field name={'background.no'}>
+                    {({ field }: FieldProps<string>) => (
+                      <>
+                        <TextField
+                          {...field}
+                          inputProps={{ maxLength: 200 }}
+                          label={t('my_page.my_profile.background')}
+                          variant="filled"
+                          multiline
+                          rows="3"
+                          placeholder={t('my_page.my_profile.field_and_background.background_placeholder')}
+                        />
+                        <Typography sx={{ alignSelf: 'end' }}>{field.value.length}/200</Typography>
+                      </>
+                    )}
+                  </Field>
+                </Box>
+                <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', mt: '1rem' }}>
+                  <Typography variant="h3">{t('my_page.my_profile.field_and_background.english')}</Typography>
+                  <Field name={'background.en'}>
+                    {({ field }: FieldProps<string>) => (
+                      <>
+                        <TextField
+                          {...field}
+                          inputProps={{ maxLength: 200 }}
+                          label={t('my_page.my_profile.background')}
+                          variant="filled"
+                          multiline
+                          rows="3"
+                          placeholder={t('my_page.my_profile.field_and_background.background_placeholder')}
+                        />
+                        <Typography sx={{ alignSelf: 'end' }}>{field.value.length}/200</Typography>
+                      </>
+                    )}
+                  </Field>
+                </Box>
+                <Box sx={{ display: 'flex', justifyContent: 'center', mt: '1rem' }}>
```

Comment:
litt space mellom knappene kanskje?
```suggestion
                <Box sx={{ display: 'flex', justifyContent: 'center', mt: '1rem', gap: "1rem" }}>
```